javascript/javascript1/week3/smart-phone.js

```diff
@@ -0,0 +1,65 @@
+const activities = [];
+const date = new Date().toLocaleDateString();
+
+// Function for adding activity into array
+function addActivity(activity, duration) {
+    if (typeof date === "string" && typeof activity === "string" && typeof duration === "number") {
+        activities.push({
+            "date": date,
+            "activity": activity,
+            "duration": duration
+        });
+        return activities;
+    } else {
+        return "Enter valid values";
+    }
+}
+
+addActivity("Youtube", 30);
+addActivity("Netflix", 22);
+addActivity("Dramacool", 40);
+console.log(activities);
+
+
+function showStatus(myArray) {
+    let totalTime = 0;
+    if (myArray && myArray === activities) {
```

Comment:
Sorry, I don't fully understand why cant' we just print list of items from `activities` array. 
Or in other words what is the sense of myArray argument for the function